cmake/Conan.cmake

```diff
@@ -7,8 +7,7 @@ macro(config_conan)
         glm/0.9.9.8
         stb/20200203
         spdlog/1.9.2
-        qt/6.2.2
-    )
+            )
```

Comment:
Please do not push your local crutches in global cmake files. It may be not compilable on other machines in theese cases.
Your issue will be solved this week

Reply:
Сори не заметил когда коммитил

Reply:
Вы из Англии?